services/common-controls.yaml

```diff
@@ -156,3 +153,39 @@ controls:
       03: |
         Confirm that backups and copies are not allowed to be
         stored in restricted regions or cloud availability zones.
+  - id: CCC.C07
+    title: Alert on non-human enumeration
+    control_family: Logging & Monitoring
+    objective: |
+      Ensure that logs and associated alerts are generated when non-human 
+      entities (e.g., automated processes) attempt to enumerate 
+      resources. This helps to detect and respond to potential malicious 
+      reconnaissance activities early.
+    threats:
+      - CCC.TH09
+    nist_csf: DE.AE-1
+    test_requirements:
+      tlp_green:
+        01: |
+          Simulate non-human enumeration activities (e.g., with service accounts) and verify that alerts are triggered in the logging system.
+        02: |
+          Confirm that logs are properly generated and accessible for review following non-human enumeration attempts.
+
+  - id: CCC.ObjStor.C08 # Enable multi-zone or multi-region data replication
```

Comment:
This has an inconsistent format compared to the other ids (CCC.CXY)

Reply:
Oof! I'll fix that in the follow-up. Thank you.